test/basic.tests.js

```diff
@@ -464,7 +465,7 @@ contract("LiquidityBridgeContract", async (accounts) => {
   it("should fail on contract call due to invalid user btc refund address", async () => {
     let rskRefundAddress = accounts[2];
     let destAddr = mock.address;
-    let data = web3.eth.abi.encodeFunctionCall(mock.abi[0], ["12"]);
```

Comment:
what is the magic number?

Reply:
I don't know, all the mock.abi[0] changed to mock.abi[1] due a library update because the first position of the abi changed to another method, I just updated the indexes so I'm not sure the meaning of the value